examples/kubernetes-es/es-sw-app.yaml

```diff
@@ -109,23 +109,33 @@ subjects:
 apiVersion: v1
 kind: Pod
 metadata:
-  name: sidious
+  name: outpost
   labels:
-    name: sidious
-    role: lord
+    app: outpost
 spec:
   containers:
-  - name: bookreader
-    image: docker.io/cilium/esclient:v1
+  - name: esclient
+    image: cilium/esindexclient:v1 
```

Comment:
Please upload the Dockefile used to build this image probably to a directory in `cilium/demo-image` and create an automated build out of there